main.go

```diff
@@ -37,58 +26,74 @@ func main() {
 		panic(err)
```

Comment:
Don't panic unless you really need the thread dump; please just print a nice message to stderr and then os.Exit(0) (or use `log.Fatalf` from stdlib)

Ditto elsewhere